### PR TITLE
DAOS-14841 tools: Fix formatting issues

### PR DESCRIPTION
**Pull request by kjacque**
- Added a shebang to a shell script.
- Skipped gofmt check on generated files.

Required-githooks: true

### Before requesting gatekeeper:

* [ ] Two review approvals and any prior change requests have been resolved.
* [ ] Testing is complete and all tests passed or there is a reason documented in the PR why it should be force landed and forced-landing tag is set.
* [x] `Features:` (or `Test-tag*`) commit pragma was used or there is a reason documented that there are no appropriate tags for this PR.
* [x] Commit messages follows the guidelines outlined [here](https://daosio.atlassian.net/wiki/spaces/DC/pages/11133911069/Commit+Comments).
* [x] Any tests skipped by the ticket being addressed have been run and passed in the PR.

### Gatekeeper:

* [ ] You are the appropriate gatekeeper to be landing the patch.
* [ ] The PR has 2 reviews by people familiar with the code, including appropriate watchers.
* [ ] Githooks were used. If not, request that user install them and check copyright dates.
* [ ] Checkpatch issues are resolved.  Pay particular attention to ones that will show up on future PRs.
* [ ] All builds have passed.  Check non-required builds for any new compiler warnings.
* [ ] Sufficient testing is done. Check feature pragmas and test tags and that tests skipped for the ticket are run and now pass with the changes.
* [ ] If applicable, the PR has addressed any potential version compatibility issues.
* [ ] Check the target branch.   If it is master branch, should the PR go to a feature branch?  If it is a release branch, does it have merge approval in the JIRA ticket.
* [ ] Extra checks if forced landing is requested
  * [ ] Review comments are sufficiently resolved, particularly by prior reviewers that requested changes.
  * [ ] No new NLT or valgrind warnings.  Check the classic view.
  * [ ] Quick-build or Quick-functional is not used.
* [ ] Fix the commit message upon landing. Check the standard [here](https://daosio.atlassian.net/wiki/spaces/DC/pages/11133911069/Commit+Comments). Edit it to create a single commit. If necessary, ask submitter for a new summary.

### CODE REVIEW

**Review by Nasf-Fan — APPROVED** [review]
Thanks for the quick fix.

**Review by ryon-jensen — APPROVED** [review]

File: src/ddb/tests/ddb_smoke_tests.sh [hunk 1]
```diff
@@ -68,8 +70,5 @@ run_cmd diff /tmp/ddb_new_value /tmp/ddb_value_dump
 
 rm -f /tmp/ddb_commands
 touch /tmp/ddb_commands
-echo "ls" >> /tmp/ddb_commands
-echo "ls [0]" >> /tmp/ddb_commands
-echo "ls [0]/[0]" >> /tmp/ddb_commands
-echo "dump_superblock" >> /tmp/ddb_commands
+{ echo "ls"; echo "ls [0]"; echo "ls [0]/[0]"; echo "dump_superblock"; } >> /tmp/ddb_commands
```
Comment (mjmac) [inline src/ddb/tests/ddb_smoke_tests.sh:73]: Heh, this is kind of nifty, hadn't encountered this construction before. It's probably fine, but might be a little more maintainable to use a heredoc, e.g.:
```
cat <<EOF>/tmp/ddb_commands
ls
ls [0]
ls [0]/[0]
dump_superblock
EOF
```

Comment (kjacque) [reply src/ddb/tests/ddb_smoke_tests.sh:73]: Agreed, this looks better. I don't seem to be able to run shellcheck locally, I'll push and see if shellcheck is happy with it.

**Review by knard-intel — APPROVED** [review]
LGTM

**Review by tanabarr — APPROVED** [review]

**Pull request merged**